.github/scripts/release_notes/SDKReleaseInfo.js

```diff
@@ -82,7 +82,7 @@ function convertToSDKReleaseInfo(releaseInfo) {
         case isIOSRelease(repoName):
             return new SDKReleaseInfo(releaseInfo, PLATFORM_ENUM.IOS, extractIOSExtensionName(repoName), tagName)
         case isAndroidRelease(repoName):
-            if (isAndroidCoreRelease(repoName)) {
+            if (isAndroidCoreRelease(repoName) || isAndroidUIRelease(repoName)) {
```

Comment:
https://github.com/adobe/aepsdk-ui-android/releases/tag/v3.0.0-notificationbuilder
Do we plan to include other artifacts/extensions to this repo? If not, I'd prefer to change the tag name for the `notificationbuilder` release to keep it consistent with other single extension repositories. 
Edge: https://github.com/adobe/aepsdk-edge-android/releases/tag/v3.0.0
cc @spoorthipujariadobe

Reply:
The repo is planned to hold multiple artifacts for different UI extension